Emulator/SPD.File.Emulator/Spd/SpdBuilder.cs

```diff
@@ -0,0 +1,362 @@
+﻿using SPD.File.Emulator.Sprite;
+using FileEmulationFramework.Lib.IO;
+using FileEmulationFramework.Lib.IO.Struct;
+using FileEmulationFramework.Lib.Utilities;
+using Microsoft.Win32.SafeHandles;
+using Reloaded.Memory.Extensions;
+using System.Text;
+using System.Runtime.InteropServices;
+
+namespace SPD.File.Emulator.Spd;
+
+public class SpdBuilder : SpriteBuilder
```

Comment:
See the comments for `SprBuilder`, since most/all required changes are pretty much the same across both files.